src/v2/Apps/Order/Mutations/CreateUserAddress.ts

```diff
@@ -20,16 +51,35 @@ export const createUserAddress = (
             ... on UserAddress {
               id
               internalID
+              addressLine1
+              addressLine2
+              addressLine3
+              city
+              country
+              isDefault
+              name
+              phoneNumber
+              postalCode
+              region
             }
             ... on Errors {
               errors {
-                code
                 message
               }
             }
           }
         }
       }
     `,
+    updater: (store, data: CreateUserAddressMutationResponse) => {
+      onAddressAdded(me, store, data)
+    },
   })
+  const errors = response?.createUserAddress?.userAddressOrErrors?.errors
+  closeModal()
```

Comment:
This `closeModal` is optional. It throws error when not there. hmm idk why typescript is not mad about it 🤔